source/shared/cpp/ObjectModel/MarkDownParser.h

```diff
@@ -18,12 +18,15 @@ class MarkDownParser
     std::string TransformToHtml();
 
     bool HasHtmlTags();
+    
```

Comment:
don't need this extra newline

Reply:
thanks; it's Xcode doing :-)

---
In reply to: [215794155](https://github.com/Microsoft/AdaptiveCards/pull/1865#discussion_r215794155) [](ancestors = 215794155)